addon/i3dio/ui/object.py

```diff
@@ -131,6 +139,100 @@ class I3DNodeObjectAttributes(bpy.types.PropertyGroup):
+        min=0,
+        max=1
+    )
+
+    linear_damping: FloatProperty(
+        name="Linear Damping",
+        description="Defines the slowdown factor for linear movement, affecting speed",
+        default=i3d_map['linear_damping']['default'],
+        min=0,
+        max=1
+    )
+
+    angular_damping: FloatProperty(
+        name="Angular Damping",
+        description="Defines the slowdown factor for angular movement, affecting spin",
+        default=i3d_map['angular_damping']['default'],
+        min=0,
+        max=1
+    )
+
+    density: FloatProperty(
+        name="Density",
+        description="Used with the shape of the object to calculate mass."
+                    "The higher the number, the heavier the object",
+        default=i3d_map['density']['default'],
+        min=0,
+        max=20
+    )
+
+    split_type: IntProperty(
+        name="Split Type",
+        description="Split type determines what type of tree it is."
+                    "For custom tree type use a number over 19",
+        default=i3d_map['split_type']['default'],
+        min=0,
+        max=200
+    )
+
+    split_type_presets: EnumProperty(
+        name="Split Type Presets",
+        description="List containing all in game tree types.",
+        items=[
+            ('0', "Custom / Manual", "Set a custom tree type or just set a tree type manually"),
+            ('1', "Spruce", "Spruce supports wood harvester"),
+            ('2', "Pine", "Pine supports wood harvester"),
+            ('3', "Larch", "Larch supports wood harvester"),
+            ('4', "Birch", "Birch doesn't support wood harvester"),
+            ('5', "Beech", "Beech doesn't support wood harvester"),
+            ('6', "Maple", "Maple doesn't support wood harvester"),
+            ('7', "Oak", "Oak doesn't support wood harvester"),
+            ('8', "Ash", "Ash doesn't support wood harvester"),
+            ('9', "Locust", "Locust doesn't support wood harvester"),
+            ('10', "Mahogany", "Mahogany doesn't support wood harvester"),
+            ('11', "Poplar", "Poplar doesn't support wood harvester"),
+            ('12', "American Elm", "American Elm doesn't support wood harvester"),
+            ('13', "Cypress", "Cypress doesn't support wood harvester"),
+            ('14', "Downy Serviceberry", "Downy Serviceberry doesn't support wood harvester"),
+            ('15', "Pagoda Dogwood", "Pagoda Dogwood doesn't support wood harvester"),
+            ('16', "Shagbark Hickory", "Shagbark Hickory doesn't support wood harvester"),
+            ('17', "Stone Pine", "Stone Pine doesn't support wood harvester"),
+            ('18', "Willow", "Willow doesn't support wood harvester"),
+            ('19', "Olive Tree", "Olive Tree doesn't support wood harvester")
+        ],
+        default='0',
+        update=lambda self, context: setattr(self, 'split_type', int(self.split_type_presets))
+    )
+
+    split_uvs: StringProperty(

```

Comment:
Could this be an [IntVector](https://docs.blender.org/api/current/bpy.props.html#bpy.props.IntVectorProperty) instead?